### DISCUSSION TITLE
Adding OTP password

### DISCUSSION THREAD

**Discussion by Danielix99** [nilsteampassnet/TeamPass#3944] [Ideas]
I would like to suggest a new feature, adding OTP will help to aggregate all of our authorization info into one single platform since we use another dedicated platform just to generate otp

**Comment by nilsteampassnet** [comment]
@Danielix99 
Thank you for your suggestion.
How do you see this?
Can you propose a use case?
This will help me to implement "something" that fits a real need.
Thanks
